### PR TITLE
PAS Status updates

### PR DESCRIPTION
**Pull request by itsbirdo**
Updates:
* Statuses for PAS proposals
* Adds requirement for PASs to be copyright free
* minor changes to formatting

### CODE REVIEW

File: README.md
Comment (hbulgarini) [inline README.md:10]: We called the support and development team, **Paseo Core Team**. Should we add this grup as a third author?

I suspect PGT stands for the Bounty curators correct?

File: README.md
Comment (hbulgarini) [inline README.md:12]: Let's use Github Labels for this. It would be good that the text says that.

@carlosala could we restrict that only members of the repos can change the labels ?

**Review by hbulgarini — CHANGES_REQUESTED** [review]

File: README.md
Comment (al3mart) [reply README.md:32]: We never introduced LEAP before. Let's keep PAS here :+1:

File: README.md
Comment (itsbirdo) [reply README.md:10]: I saw "Paseo Governance Team" referenced somewhere else - happy to go with whatever and set a standard.
IMO there are two groups
1. Core team that have sudo access for changing the chain - this group should be in charge of governance of the network and changes etc
2. Curator team that are approving and reviewing payments

File: README.md
Comment (itsbirdo) [reply README.md:12]: Labels are good - I guess the point was when you change the status it has to be a PR change, so that it needs to be approved and there is more of an audit trail.

**Comment by itsbirdo** [comment]
@hbulgarini waiting on your review

**Review by al3mart — DISMISSED** [review]

File: README.md
Comment (educlerici-zondax) [reply README.md:12]: But Github PR status are not enough to track these status?
we would have: 
- Open: Proposed and under discussion PAS
- Merged: Approved and merged PAS
- Closed: Dissaproved PAS

**Comment by hbulgarini** [comment]
> @hbulgarini waiting on your review

@itsbirdo i'm still not convinced about the status discussion and Edu also has a different approach than me:

https://github.com/paseo-network/paseo-action-submission/pull/36/files#r1511586124

**Comment by educlerici-zondax** [comment]
is there an issue created to link this PR?

**Comment by educlerici-zondax** [comment]
@itsbirdo all the bulletpoints you added in the issue description are done. can we close this issue?

**Comment by educlerici-zondax** [comment]
@itsbirdo can we resolve the conflicts and close this PR?

**Review by hbulgarini — APPROVED** [review]

**Pull request merged**
